.travis.yml

```diff
@@ -15,7 +14,7 @@ install:
 - conda config --add channels bokeh
 - conda config --add channels conda-forge
 - conda info -a
-- conda install conda-build nodejs selenium jupyterlab notebook geckodriver firefox
+- conda install conda-build nodejs jupyterlab notebook
```

Comment:
We didn't need selenium now when we are using the browser_check script instead of the selenium_check script, and I think geckodriver / firefox is meant for selenium.